infra/aws/terraform/prow-build-cluster/resources/node-problem-detector/node-problem-detector.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         command:
         - /node-problem-detector
         - --logtostderr
-        - --config.system-log-monitor=/config/kernel-monitor.json,/config/containerd-monitor.json
+        - --config.system-log-monitor=/config/kernel-monitor.json,/config/docker-monitor.json
```

Comment:
Why is this changed? It is supposed to be containerd-monitor and it's also that way in the ConfigMap.

Reply:
Can we fix update.bash script to handle this properly?

Reply:
Jan introduced a dedicated script for updating node-problem-detector.